WatchGuard patches to sources for bzip2-1.0.6 as of
Mon May  9 13:48:10 PDT 2022
The patches shown here have been applied to source .tar.gz 
files supplied with the WatchGuard Open Source Archive.

==========================================================================
--- bzip2-1.0.6/Makefile-libbz2_so.orig	2022-05-09 13:48:09.325522875 -0700
+++ bzip2-1.0.6/Makefile-libbz2_so	2022-05-09 13:48:09.445518004 -0700
@@ -43,6 +43,15 @@
 clean: 
 	rm -f $(OBJS) bzip2.o libbz2.so.1.0.6 libbz2.so.1.0 bzip2-shared
 
+
+install: $(BINS)
+	if ( test ! -d $(PREFIX)/lib ) ; then mkdir -p $(PREFIX)/lib ; fi
+	if ( test ! -d $(PREFIX)/bin ) ; then mkdir -p $(PREFIX)/bin ; fi
+	cp -f libbz2.so.1.0.6 $(PREFIX)/lib
+	chmod a+r $(PREFIX)/lib/libbz2.so.1.0.6
+	cp -f bzip2-shared $(PREFIX)/bin
+	chmod a+r $(PREFIX)/bin/bzip2-shared
+
 blocksort.o: blocksort.c
 	$(CC) $(CFLAGS) -c blocksort.c
 huffman.o: huffman.c
